Jellyfin.Api/Controllers/MediaInfoController.cs

```diff
@@ -302,27 +302,12 @@ public async Task<ActionResult> CloseLiveStream([FromQuery, Required] string liv
         /// </summary>
         /// <param name="size">The bitrate. Defaults to 102400.</param>
         /// <response code="200">Test buffer returned.</response>
-        /// <response code="400">Size has to be a numer between 0 and 10,000,000.</response>
         /// <returns>A <see cref="FileResult"/> with specified bitrate.</returns>
         [HttpGet("Playback/BitrateTest")]
         [ProducesResponseType(StatusCodes.Status200OK)]
-        [ProducesResponseType(StatusCodes.Status400BadRequest)]
-        [Produces(MediaTypeNames.Application.Octet)]
         [ProducesFile(MediaTypeNames.Application.Octet)]
-        public ActionResult GetBitrateTestBytes([FromQuery] int size = 102400)
+        public ActionResult GetBitrateTestBytes([FromQuery][Range(1, 100_000_000, ErrorMessage = "The requested size must be greater than 0 and less than 100,000,000")] int size = 102400)
```

Comment:
The parameters of Range are treated as inclusive.
Also, if I understand it correctly, you can use `{1}` and `{2}` instead of hardcoded values in the error message.

```suggestion
        public ActionResult GetBitrateTestBytes([FromQuery][Range(1, 100_000_000, ErrorMessage = "The requested size must be greater than 0 and equal to or less than 100,000,000")] int size = 102400)
```